native/cocos/engine/Engine.cpp

```diff
@@ -82,6 +82,8 @@ namespace cc {
 Engine::Engine() {
     _scheduler = std::make_shared<Scheduler>();
     FileUtils::getInstance()->addSearchPath("Resources", true);
+    FileUtils::getInstance()->addSearchPath("data", true);
```

Comment:
Is it needed?

Reply:
`Resources/data` is not needed and removed